From 4d3aa300333a97e653deb085b871c3131077cf30 Mon Sep 17 00:00:00 2001
From: Kondapally Kalyan <kalyan.kondapally@intel.com>
Date: Sat, 31 Aug 2013 03:01:13 +0300
Subject: [PATCH] Chromium: Add gl changes for Wayland

This patch should not be upstreamed.
Ideally, we would need an api to query for supported
EGLConfig.
---
 ui/gl/gl_surface_egl.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gl/gl_surface_egl.cc b/ui/gl/gl_surface_egl.cc
index 5f4edf4..b99a0cf 100644
--- a/ui/gl/gl_surface_egl.cc
+++ b/ui/gl/gl_surface_egl.cc
@@ -140,7 +140,7 @@ bool GLSurfaceEGL::InitializeOneOff() {
     EGL_GREEN_SIZE, 8,
     EGL_RED_SIZE, 8,
     EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
-    EGL_SURFACE_TYPE, EGL_WINDOW_BIT | EGL_PBUFFER_BIT,
+    EGL_SURFACE_TYPE, EGL_WINDOW_BIT,
     EGL_NONE
   };
 
-- 
1.7.9.5

